### PR TITLE
Build New Gitcoin Landing page

### LINKED ISSUE #1104: Build New Gitcoin Landing Page
**Extra 10% Tip for completion by May 18th!**

As potential funder or contributor, I want to know what Gitcoin is and understand the problems it can solve for my scenario.

# Description
The current Gitcoin.co landing page needs updating with new content and allow for more information to entice our funders.

# Assumptions
Build this page from scratch (please do not retrofit any styles or functionality from the current landing page).

# Definition of Done

- [ ] Build new landing page from scratch (it's ok to start a new css style sheet).
- [ ] Update all fonts to Muli except the top banner (please check sizing on all fonts).

- [ ] This page should have a few parallax elements (please work with @pixelantdesign).
- [ ] Gitcoin tree and blue box
- [ ] Big Robot and smoke

- [ ] The top nav should follow a user down the page so that fund issue is persistent during the entire 
experience.

- [ ] Use font awesome [Ethereum icon](https://cdn.worldvectorlogo.com/logos/ethereum-cash.svg) here for the fund button - please make white (vs, the one provided in the design).

This page should be built using SVG animations (small animations)
- [ ] slowly animate windmil blades to spin
- [ ] animate movement on the lines on the globe
- [ ] make the leaves in the pot rotate slightly to the right

How it works module
- [ ] Allow for toggling between Funder and Contributor
- [ ] Supporting content should display according to the button selected.
In the mockup the Contributor toggle and content is shown.

Videos
- [ ] How to use Gitcoin in 30 seconds should play in a lightbox
- [ ] Gitcoin in 30 seconds should play in a lightbox

Case Studies
- [ ] Each case study will link to an article to be provided
- [ ] See all case studies will link to a case studies page (part of another ticket)

Who's Using Gitcoin?
Clicking on the icon below the photo will change the content above it (Photo and testimonial).

Press
- [ ] Each logo should hover in color, clicking on the logo should link to each article (please reference the current landing page.

Footer
- [ ] Format of the footer has been updated.
- [ ] Please animate the saturn planet to hover and slightly move (nice to have if there is time at the end).
- [ ] Reference footer links in the current landing page.

Support Responsive Views

- [ ] - Leverage type and button styles defined in the sketch file included

Work with @PixelantDesign on any outstanding questions

# Reviewers
@PixelantDesign @owocki @mbeacom

# References
gitcoin landing final

Nave Follow State
follow nav

Sketch File
[Gitcoin Landing Final.zip](https://github.com/gitcoinco/web/files/1983346/Gitcoin.Landing.Final.zip)

### PR DESCRIPTION
**Pull request by Smona**
##### Description
Creates a new Gitcoin landing page

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
ui

##### Refers/Fixes
Fixes #1104

### CODE REVIEW

**Comment by Smona** [comment]
@PixelantDesign @owocki @thelostone-mc All Done! :champagne: 

The only remaining task is to add How It Works section copy for funders, and provide correct anchor destination for some of the links.

I apologize that this ran a little long; life got real full all of a sudden! But I think the results are worth the wait ^.^

File: app/assets/v2/css/landing_page.css [hunk 1]
```diff
@@ -20,6 +20,9 @@
   right: calc(5vw - 50px);
   max-width: 30vw;
 }
+#gc-tree {
```
Comment (thelostone-mc) [inline app/assets/v2/css/landing_page.css]: might wanna add a new line there ^_^

**Comment by thelostone-mc** [comment]
#### Screenshot 

screencapture-localhost-8000-2018-05-29-02_08_14 1


#### GIF 

x1

---

#### Questions for Gitcoin Core

@PixelantDesign  Could we reduce the tree size ? it feels super huge (Any thoughts @Smona  ? ) 

@PixelantDesign / @vs77bb / @owocki Could we get colored logos of our press ?  :P 

screen shot 2018-05-29 at 2 19 33 am

---

@Smona Love it :D 
PS:  nice work on the responsive design! Left a few comments  (Sorry for the long post )

- Add padding (top and bottom ) to the button like in the design 

screen shot 2018-05-29 at 2 25 55 am


- Could we make the cards taller and thinner like in the design ?

screen shot 2018-05-29 at 2 23 24 am


- Add a margin top and bottom to the button to make it more spaced out (like in the design )

screen shot 2018-05-29 at 2 15 06 am


- Could we reduce the font size for the steps (like in the design) + make the button taller ( more padding top and bottom )

screen shot 2018-05-29 at 2 13 23 am


- Make the button wider + Links are greyed out in the design (not sure why ) Keepingit blue makes sense though. Your thoughts @Smona  ?
screen shot 2018-05-29 at 2 21 45 am


- Add more margin at left and right side for the cards (like in the design) ?

screen shot 2018-05-29 at 2 20 35 am


- Footer links seem lighter in the design ( reduce the font weight and update the color to match the design ) ? 

screen shot 2018-05-29 at 2 17 21 am


Sorry for the long post  😅 

@SaptakS  / @mbeacom anything to add  ? ^_^

**Comment by owocki** [comment]
looking good!

feedback:
1. oss tree is offset from the purple module below it in the designs.  it touches the puprple module in the implementation
2. can we get screenshots of how it looks responsive?
3. h1 in the hero is capitalized but is not in the design
4. right below hero, the green module, the links are not the color they are in the design
5. the links in the case studies are not the right colors.. and looks like the robot is not proper (though this might be an artifact of the screenshto)
6. testimonials dont match the design (see screenshot => http://bits.owocki.com/0u3M1J2v0Q0y/Screen%20Shot%202018-05-28%20at%208.35.09%20PM.png)

@thelostone-mc  colored press logos are here https://github.com/gitcoinco/web/tree/master/app/assets/v2/images/press

**Comment by thelostone-mc** [comment]
@Smona could you also update the press images to the colored ones ^_^ 

Just out of curiosity , what's your screen size @owocki ? The testimonals spacing was fine on my 15inch. :P

**Comment by thelostone-mc** [comment]
@Smona Could you squash the commits into a single commit or 2 & also resolve the conflicts ? 
Tried rebasing it but kept encountering multiple conflicts along the way after running `git rebase --continue`

**Comment by mbeacom** [comment]
Merged in: https://github.com/gitcoinco/web/pull/1377

**Pull request closed without merging**